app/javascript/shared/components/plateScanValidators.js

```diff
@@ -375,6 +375,28 @@ const checkPlateWithSameReadyLibrarySubmissions = (cached_submission_ids) => {
   }
 }
 
+// Returns a validator that ensures that the scanned plate does not have an unacceptable plate purpose.
+// plate: The current plate being scanned
```

Comment:
document the type and expected contents of the `acceptable_purposes` parameter?

Reply:
Done